quickmpc/qmpc_server.py

```diff
@@ -129,12 +130,18 @@ def __futures_result(
         return is_ok, response
 
     @staticmethod
-    def __stream_result(stream: Iterable) -> Dict:
+    def __stream_result(stream: Iterable, job_uuid: str, party: int,
+                        path: Union[str, None]) -> Dict:
```

Comment:
I am happier with `Optional[**]` than `Union[**, None]`.

Reply:
@mdonaka 
Thanks for the review!
I fixed it with a [c9ef2d4](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/25/commits/c9ef2d401ac7faa8c02350dfb88bfa5f39ce4a41).